src/Vcpkg.cmake

```diff
@@ -108,7 +108,24 @@ macro(run_vcpkg)
   list(APPEND VCPKG_FEATURE_FLAGS "versions")
   set(CMAKE_TOOLCHAIN_FILE
       ${_vcpkg_args_VCPKG_DIR}/scripts/buildsystems/vcpkg.cmake
-      CACHE STRING "vcpkg toolchain file")
+      CACHE STRING "vcpkg toolchain file" FORCE)
```

Comment:
Should this be forced?

Reply:
In `enable_cross_compiler()` I'm already setting a toolchain (as default), so if you are not using vcpkg, this should be the default, otherwise vcpkg overrides it and sets `VCPKG_CHAINLOAD_TOOLCHAIN_FILE` in addition.

_but this case need also more testing_